pyproject.toml

```diff
@@ -226,6 +226,8 @@ pydantic = ">=1.7.4, <3"
 # needed.
 setuptools_rust = ">=1.3"
 
+uvloop = ">=0.19.0"
```

Comment:
Note that the reactor tick timing stuff won't work with uvloop. I think I tried to support it briefly and then decided we probably weren't going to use it...

Reply:
I think the asyncio reactor you end up doing some bookkeeping twice, which is annoying. I'm kind of surprised it is that much of a difference though, I wish we had a good benchmark for running this through instead of sytest.